CONTRIBUTING.md

```diff
@@ -7,3 +7,11 @@ Some guidelines you should follow when submitting a PR:
 * We use [`generate-changelog`](https://github.com/lob/generate-changelog) for automatic changelog generation whenever we deploy, so please follow the commit message format: `feat(info): add animations when toggling`.
 * Ideally, your PR has one commit in it. If you you need to add another one because of a typo or refactor, you should squash your commits together. This makes the changelog and commit history much more managable to go through. If you need help squashing them, we can always help you out!
 * Before submitting the PR, you should make sure that your code passes the linter. Running `npm run lint` locally will speed things up because the TravisCI build will fail if the linting doesn't pass.
+
+## Shrinkwrap
+
+Since this project is built using Node and NPM, we have a `package.json` to list all of the top-level dependencies that we use. Since the version numbers in there can leave a lot up for interpretation, we also have an [`npm-shrinkwrap.json` file](https://docs.npmjs.com/cli/shrinkwrap) to lock down exact versions of direct dependencies and the versions of nested dependencies. One issue with the shrinkwrap though is that [it can make any optional dependencies required](https://github.com/npm/npm/issues/2679). One such example of this is the [`fsevents`](https://www.npmjs.com/package/fsevents) module. This is an optional dependencies that is only able to be build on Mac OS X. So if this module becomes required, `npm i` on any machine other than a Mac will fail.
+
+Since there currently isn't a fix for this just yet, we've employed [a workaround](https://github.com/npm/npm/issues/2679#issuecomment-150084700) of just deleting the optional dependency from the shrinkwrap and then manually listing it under `optionalDependencies`. The problem with this is that while it's easy to take it out, it's difficult to keep it out. Because TravisCI doesn't support running a build on Windows to ensure functionality, we have to get a little manual and just make sure `fsevents` isn't included in the shrinkwrap. Even though this isn't going to be exhaustive, it's at least a start.
```

Comment:
maybe italicize "take" and "keep"? i had to re-read a few times to see what you meant.